### PR TITLE
Added German translation to Quick Pad

### PR DESCRIPTION
**Pull request by MuMDevelopment**

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@MuMDevelopment Can you target the `develop` branch for this?

File: src/QuickPad.UI/QuickPad.UI/MultilingualResources/QuickPad.UI.ar-EG.xlf [hunk 1]
```diff
@@ -854,14 +854,14 @@
...
         </trans-unit>
         <trans-unit id="WelcomeDialogPar4.Text" translate="yes" xml:space="preserve">
           <source></source>
           <target state="needs-review-translation">• لقد اصلحنا إلاغلاق المفاجئ الذي كان يعاني منه بعض المستخدمين عند حفظ مستنداتهم.</target>
-          <note from="MultilingualBuild" annotates="source" priority="2">Used on Welcome Dialog</note>
           <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
+          <note from="MultilingualBuild" annotates="source" priority="2">Used on Welcome Dialog</note>
         </trans-unit>
         <trans-unit id="AskForReviewDialog.Title" translate="yes" xml:space="preserve">
           <source>Review Quick Pad</source>
```
Comment (yaichenbaum) [inline src/QuickPad.UI/QuickPad.UI/MultilingualResources/QuickPad.UI.ar-EG.xlf:867]: Can you revert the changes to the ar-EG resource file?

**Comment by MuMDevelopment** [comment]
I'm sorry. I have not modified the ar-EF file. But after forking the project and opening with Visual Studio, it says that there are modifications to that file. Don't know what to do?
As far as I can see, the changes in the ar-EG file are just some notes...

**Comment by yaichenbaum** [comment]
@MuMDevelopment I am not sure why it was tagged as changed since the file looks identical to the previous version, thank you for your contribution. Make sure to subscribe to #163 in order to be notified when new strings are added.
